### DISCUSSION TITLE
Watch option - `nuxt.config`?

### DISCUSSION THREAD

**Discussion by ivodolenc** [nuxt/nuxt#16126] [Questions]
Hi, is there a `watch` option in Nuxt 3 like there was in [Nuxt 2](https://nuxtjs.org/docs/configuration-glossary/configuration-watch/)? If yes, is there any code example?

I ask because I have a case with a custom module, when the user makes changes to a files inside a custom directory (dir is at the project's root), it would be great if server can be automatically restart thus improve the development experience (no need for manually restart).

So, I would like to manipulate the `watch` option via module. Any advice?

**Comment by aliuq** [comment]
Nobody answer here

**Comment by Baroshem** [comment]
What about chokidar or general watchers configuration option in nuxt config? 

https://nuxt.com/docs/api/configuration/nuxt-config/#chokidar

**Reply by aliuq** [reply]
This doesn't look like you can add an outside file
